misc/Dockerfile

```diff
@@ -142,10 +142,6 @@ RUN \
     "y" \
     # Admin password
     "nbusr123" \
-    # Facebook OAuth Client ID (default)
-    "" \
-    # Facebook OAuth Secret (default)
-    "" \
```

Comment:
@viliambalaz Rovnako to treba vyhodit aj z `.github/workflows/main.yml`. Inac haluz, ze zbehli testy v CI.

Reply:
:ok: